package.json

```diff
@@ -37,19 +37,133 @@
     "url": "https://github.com/terrestris/react-geo-baseclient/issues"
   },
   "homepage": ".",
-  "dependencies": {},
   "scripts": {
-    "add": "lerna add",
-    "bootstrap": "lerna bootstrap --no-ci",
-    "test": "lerna run test --scope *baseclient*",
-    "start-static": "lerna run start-static --stream",
-    "start-shogun2": "lerna run start-shogun2 --stream",
-    "build:static": "cd packages/baseclient/ && npm run build:static && cd ../../",
-    "build:shogun2": "cd packages/baseclient/ && npm run build:shogun2 && cd ../../",
-    "build:ts": "lerna run build:ts --stream",
-    "postinstall": "npm run bootstrap && cd packages/react-geo && npm run build:dist && cd ../baseclient-state && npm run build:ts && cd ../baseclient-components && npm run build:ts"
+    "start:static": "webpack-dev-server --config config/webpack.static.config.js --hot",
+    "start:shogun2": "env-cmd .shogunrc webpack-dev-server --config config/webpack.shogun2.config.js --hot",
```

Comment:
Is this successfully tested?

Reply:
No. Not yet. I think i'll tackle this with @mholthausen on monday. There might be a follow up PR incoming.